albumentations/augmentations/geometric/functional.py

```diff
@@ -1,19 +1,15 @@
 import cv2
 import math
 import numpy as np
+import six.moves as sm
```

Comment:
Can we drop six package? It serves to smoothen transition form python2 to python3. And the transition seems to be trivial: `sm.xrange -> range`.

Reply:
Done